server/test/api/inventory.js

```diff
@@ -0,0 +1,202 @@
+  });
+
+  it('PUT /inventory/:uuid/metadata update an existing inventory metadata', () => {
+    return agent.put('/inventory/' + metadata.uuid + '/metadata')
+      .send(metadataUpdate)
+      .then((res) => {
+        // NOTE: Returned data are from the /inventory/:uuid/metadata API
+        // these data are not sended by the test but come from join with other table :
+        // label, groupNmae, type, unit
+        expect(res.body[0].uuid).to.be.equal(metadataUpdate.uuid);
+        expect(res.body[0].code).to.be.equal(metadataUpdate.code);
+        expect(res.body[0].text).to.be.equal(metadataUpdate.label);
+        expect(res.body[0].price).to.be.equal(metadataUpdate.price);
+      })
+      .catch(helpers.handler);
+  });
+
+  // ========================== inventory groups ==============================
+
+  // create inventory group
+  it('POST /inventory/group create a new inventory group', () => {
+    return agent.post('/inventory/groups')
+      .send(inventoryGroup)
+      .then((res) => {
+        helpers.api.created(res);
+        expect(res.body.uuid).to.be.equal(inventoryGroup.uuid);
+      })
+      .catch(helpers.handler);
+  });
+
+  // update inventory group
+  it('PUT /inventory/group/:uuid updates an existing inventory group', () => {
+    return agent.put('/inventory/groups/' + inventoryGroup.uuid)
+      .send(updateGroup)
+      .then((res) => {
+        let group = res.body[0];
+        updateGroup.uuid = inventoryGroup.uuid;
+        expect(group).to.contain.all.keys(Object.keys(updateGroup));
+        for(var i in group) {
+          expect(group[i]).to.be.equals(updateGroup[i]);
+        }
+      })
+      .catch(helpers.handler);
+  });
+
+  // list of inventory groups
+  it('GET /inventory/group returns list of inventory groups', () => {
+    return agent.get('/inventory/groups')
+      .then((res) => {
+        countInventoryGroups = res.body.length;
+        helpers.api.listed(res, countInventoryGroups);
+      })
+      .catch(helpers.handler);
+  });
+
+  // detailS of inventory groups
+  it('GET /inventory/group returns details of an inventory group', () => {
+    return agent.get('/inventory/groups/' + inventoryGroup.uuid)
+      .then((res) => {
+        let group = res.body[0];
+        expect(group).to.contain.all.keys(Object.keys(inventoryGroup));
+        // compare value to the last update of our request
+        for(var i in group) {
+          expect(group[i]).to.be.equals(updateGroup[i]);
+        }
+        helpers.api.listed(res, 1);
+      })
+      .catch(helpers.handler);
+  });
+
+  // ========================== inventory types ===============================
+
+  // create inventory type
+  it('POST /inventory/types create a new inventory types', () => {
+    return agent.post('/inventory/types')
+      .send(inventoryType)
+      .then((res) => {
+        inventoryType.id = res.body.id;
+        helpers.api.created(res);
+        expect(res.body.id).to.be.equal(inventoryType.id);
```

Comment:
This test will always succeed.  You are doing this:
```js
inventoryType.id = res.body.id;
expect(res.body.id).to.be.equal(inventoryType.id);
```
The `expect()` will always be true, unless something is very wrong with JavaScript.  It would be better to test that the id exists, or not perform this test as all...